tests/python/rest_api/test_labels.py

```diff
@@ -13,10 +13,11 @@
 from cvat_sdk import exceptions, models
 from cvat_sdk.api_client.api_client import ApiClient, Endpoint
 from cvat_sdk.core.helpers import get_paginated_collection
+from dateutil.parser import parse as parse_datetime
```

Comment:
Should probably use `isoparse`, to make sure the API outputs are in the expected format.

Reply:
I don't think it adds anything to the test. These tests are not focused on a specific output format.

Reply:
Sure, but if they happen to catch an issue with the date output format, that would be useful, no? And it should be a trivial change.

Reply:
Ok